### ISSUE TITLE
ORD validation failing in Unified Customer Landscape

### ISSUE THREAD

**Issue by StefanHenke** [cap-js/ord#102]
I have created a ORD document for a CAP Java application with a very simple CDS model using the @cap-js/ord plugin.

UCL complains about a few things:

**sap-ord-consumption-bundle-for-inbound (warning)** : An API Resource of direction `inbound` or `mixed` SHOULD have at least one consumption bundle assigned.
**sap-ord-schema (error)**: "entityTypeTargets" property must not have fewer than 1 items.
**sap-ord-schema-x-recommended**: Property `lastUpdate` is missing.

![image](https://github.com/user-attachments/assets/a537b249-258c-4269-bf21-ee745d971e47)

My expectation is that either the generated document is accepted by UCL without issues or the ord plugin already makes me aware of potential errors already. Although I think that the mentioned two issues more look like incorrectly generated documents.

The generated ORD document:
[test.json](https://github.com/user-attachments/files/17945885/test.json)

CDS Model:
```
service EbMtEmitter {
    action sendMsg();

    event MyEvent {
        key id : Integer;
        message : String;
    }

}
```

**Comment by aramovic79** [comment]
Hi @StefanHenke 
I've checked locally the result of the ORD Plugin "execution" for the given service definition and the observed the following:

1. **Warning** `... SHOULD have at least one consumption bundle assigned` is shown because there is no property `partOfConsumptionBundles` in none of the `apiResources`. The implementation is not yet in place and it will follow after internal discussion and clarification. This field/property is optional in ORD document, as you can see [here](https://pages.github.tools.sap/CentralEngineering/open-resource-discovery-specification/spec-v1/interfaces/document#api-resource_partofconsumptionbundles). Hence, although the generated ORD document does not contain this property/field, it is still a valid ORD document;
2. **Error** `"entityTypeTargets" property must not have fewer than 1 items.` is indeed a bug that has to be fixed. In case of "your" service definition there is no entity(and no entity annotated by `@ODM.entityName: 'odm.bookshop.Authors'` annotation) which `ORD Plugin` "uses" for collecting the information about target entities. This is kind-of corner case we didn't cover. The correct result in this case is to fully omit property `entityTypeMappings`. This fix will be included in the version 1.3.0.
3. **Warning** `Property lastUpdate is missing.` is reported because you are probably using version <1.2.0 because we fixed this bug in version 1.2.0. 
I've created a follow-up issue: https://github.com/cap-js/ord/issues/103. 
Please leave a comment if you disagree.
cc: @Fannon @swennemers @zongqichen

**Comment by aramovic79** [comment]
@StefanHenke : This is the result of validation of the ORD document "generated" locally for the given service definition(1 error and 1 warning):
![image](https://github.com/user-attachments/assets/90c70383-a963-466c-9190-bf6caf7e9c3a)

**Comment by aramovic79** [comment]
his issue will be closed by #103 once it is resolved.
